src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTests.cs

```diff
@@ -0,0 +1,312 @@
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Maestro.Client;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Octokit;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Maestro.ScenarioTests
+{
+    public class MaestroScenarioTests : IAsyncLifetime
+    {
+        private TestParameters _parameters;
+
+        private readonly ITestOutputHelper _output;
+        private readonly Random _random = new Random();
+
+        public IMaestroApi MaestroApi => _parameters.MaestroApi;
+
+        public GitHubClient GitHubApi => _parameters.GitHubApi;
+
+        public MaestroScenarioTests(ITestOutputHelper output)
+        {
+            _parameters = null!;
+            _output = output;
+        }
+
+        public async Task InitializeAsync()
+        {
+            _parameters = await TestParameters.GetAsync(_output);
+        }
+
+        public Task DisposeAsync()
+        {
+            return Task.CompletedTask;
```

Comment:
Why be dispose that does nothing?

Reply:
This is the IAsyncLifetime thingy that xunit has. I needed it for InitializeAsync, but I don't have any dispose to do.

Reply:
Actually... I do have something to dispose. I'll add it.